script/vsts/platforms/templates/preparation.yml

```
@@ -20,9 +20,17 @@ steps:
     condition: eq(variables['Agent.OS'], 'Linux')
 
   # Common
+  - bash: |
+      if [ "${BUILD_ARCH}" = "x86" ]; then \
+        echo "##vso[task.setvariable variable=SHOULD_FORCE_32_BIT_NODE]true"
+      fi
```

This should use pwsh to be consistent with all the other scripts.

```suggestion
  - pwsh: |
      if ($env:BUILD_ARCH -eq "x86") {
        echo "##vso[task.setvariable variable=SHOULD_FORCE_32_BIT_NODE]true"
      }
```

---

I suppose this is the right thing to do.

If I could go back in time and standardize on `bash`, I would. I'm much more familiar with it, and most Atom maintainers don't seem to use Windows very much. So the chance is they are better with `bash` than Powershell. And I personally am familiar with `bash`, not Powershell. Somehow when we first discussed this I didn't think `bash` was an option, or that it wouldn't work on Windows, but I see now that it does. 🤷 So anyhow, some time in the future I would personally like it if we moved everything to `bash`... but it's better for now to be consistent.